scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,181 @@
+  -H "Authorization: Bearer ACCESS-TOKEN" \
+  -H 'accept: application/json' \
+  -H 'content-type: application/json' \
+  --data-raw '{"query":"query getAppAcceleratorRuns { appAcceleratorRuns(first: 250){ nodes { guid name namespace timestamp } pageInfo{ endCursor hasNextPage } } }"}' | jq .
+```
+
+Where:
+
+- `ACCESS-TOKEN` is the AMR GraphQL access token
+- `/tmp/graphql-ca.crt` is the file location containing the AMR GraphQL CA certificate
+
+You can use this to write and execute your own GraphQL queries to fetch data from the AMR.
+
+This section uses cURL to query the AMR GraphQL endpoint, but you can use other similar tools to access the endpoint
+and provide them with the AMR GraphQL access token and CA certificate.
+
+## <a id='query-app-accelerator-runs'></a> Querying for AppAcceleratorRuns (alpha)
+
+This section tells you about GraphQL query arguments, and lists the fields available for `AppAcceleratorRuns` and `AppAcceleratorFragments`.
+
+### <a id='app-accelerator-runs-query-args'></a> AppAcceleratorRuns query arguments
+
+(Optional) You can specify the following supported arguments when querying for `AppAcceleratorRuns`.
+
+- `query`: expects an object that specifies additional arguments to query. The following arguments are supported in this query object:
+
+- `guid`: UID identifying the run, as a `String` value. Each AppAcceleratorRun is automatically assigned a UID.
+
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{guid: "d2934b09-5d4c-45da-8eb1-e464f218454e"})
+  ```
+
+- `source`: string representing the client used to run the accelerator. Supported values include `TAP-GUI`, `VSCODE`, and `INTELLIJ`.
+
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{source: "TAP-GUI"})
+  ```
+
+- `username`: string representing the user name of the person who runs
+the accelerator, as captured by the client UI.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{username: "homer.simpson"})
+  ```
+
+- `namespace` and `name`: strings representing the accelerator that
+was used to create an application.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{name: "tanzu-java-web-app"})
+  ```
+
+- `appAcceleratorRepoURL`, `appAcceleratorRevision`, and `appAcceleratorSubpath`: actual location in VCS (Version Control System) about the accelerator sources used.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{
+    appAcceleratorRepoURL: "https://github.com/vmware-tanzu/application-accelerator-samples.git",
+    appAcceleratorRevision: "v1.6"
+  })
+  ```
+
+- `timestamp`: string representation of the exact time the accelerator ran. You can query for runs that happened `before` or `after` a particular instant:
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query: {timestamp: {after: "2023-10-11T13:40:46.952Z"}})
+  ```
+
+### <a id='app-accelerator-runs-fields'></a> AppAcceleratorRuns fields
+
+You can choose the following fields to return in the GraphQL query.
```

Comment:
The terms fields is used here, but in line 90 above arguments is used?
Which is the correct term or are they intended to mean different things here?

Reply:
They are intended to mean different things
Arguments are something a User can pass into the query.
Fields are what data a user wants returned from the query

e.g. show me all the sizes (field) for tshirts that are blue in color (argument)